src/Modules/CalcOffence.lua

```diff
@@ -2883,6 +2883,10 @@ function calcs.offence(env, actor, activeSkill)
 			local rateMod = calcLib.mod(skillModList, cfg, "BleedFaster") + enemyDB:Sum("INC", nil, "SelfBleedFaster")  / 100
 			globalOutput.BleedDuration = durationBase * durationMod / rateMod * debuffDurationMult
 			local bleedStacks = (output.HitChance / 100) * (globalOutput.BleedDuration / output.Time) / maxStacks
+			local activeTotems = env.modDB:Override(nil, "TotemsSummoned") or skillModList:Sum("BASE", skillCfg, "ActiveTotemLimit", "ActiveBallistaLimit")
+			if skillFlags.totem then
```

Comment:
This `if` is not really necessary given that `local activeTotems` will be 1 when no totems, or a valid value if totems.

Reply:
This makes sense when reading the code, but in practice if a totem build had other sources of maximum totems, they'll still apply to skills that aren't supported by totems without it